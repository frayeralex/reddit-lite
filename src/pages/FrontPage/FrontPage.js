import React from 'react';
import { Layout } from 'templates';
import './FrontPage.scss';

export class FrontPage extends React.Component {
  render() {
    return (
      <Layout>
        <div className="FrontPage">FrontPage</div>
      </Layout>
    );
  }
}

export default FrontPage;
