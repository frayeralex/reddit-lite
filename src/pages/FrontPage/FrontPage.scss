.FrontPage {
  text-align: center;

  .post-list {
    padding-top: 20px;

    .Post {
      margin-bottom: 20px;
      border-radius: 3px;

      &:last-child {
        margin-bottom: 0;
      }
    }
  }
}
