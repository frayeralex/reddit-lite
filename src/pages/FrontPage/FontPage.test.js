import React from 'react';
import { FrontPage } from './FrontPage';

describe('<FrontPage/>', function() {
  it('should render without crushing', function() {
    const component = shallow(<FrontPage />);
    expect(component).toMatchSnapshot();
  });
});
