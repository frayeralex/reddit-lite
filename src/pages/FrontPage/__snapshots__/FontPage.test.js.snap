// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FrontPage/> should render without crushing 1`] = `
<Layout>
  <div
    className="FrontPage"
  >
    FrontPage
  </div>
</Layout>
`;
